warp10/src/main/java/io/warp10/continuum/gts/GTSEncoder.java

```diff
@@ -496,10 +508,10 @@ public synchronized int addValue(long timestamp, long location, long elevation,
         if (Math.abs(elevation) < (1L << 48)) {
           locElevFlag |= FLAGS_ELEVATION_ZIGZAG;
         }
-        noDeltaMetaElevation = false;
       }
     } else {
       lastElevation = GeoTimeSerie.NO_ELEVATION;
+      validLastElevation = false;
```

Comment:
`lastElevation = GeoTimeSerie.NO_ELEVATION;` is unnecessary since `validLastElevation` is set to `false`.